sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -0,0 +1,190 @@
+WITH unioned AS (
+  SELECT
+    *,
+    'Fenix' AS normalized_app_name
+  FROM
+    fenix.clients_last_seen_joined
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  SELECT
+    *,
+    'Firefox iOS' AS normalized_app_name
+  FROM
+    firefox_ios.clients_last_seen_joined
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  SELECT
+    *,
+    'Focus iOS' AS normalized_app_name
+  FROM
+    focus_ios.clients_last_seen_joined
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  SELECT
+    submission_date,
+    normalized_channel,
+    client_id,
+    udf_js.sample_id(client_id) AS sample_id,
+    days_since_seen,
+    days_since_created_profile,
+    NULL AS days_since_seen_session_start,
+    NULL AS days_since_seen_session_end,
+    days_seen_bits,
+    days_created_profile_bits,
+    first_seen_date AS first_run_date,
+    durations,
+    NULL AS days_seen_session_start_bits,
+    NULL AS days_seen_session_end_bits,
+    os AS normalized_os,
+    osversion AS normalized_os_version,
+    NULL AS android_sdk_version,
+    locale,
+    city,
+    country,
+    app_build_id AS app_build,
+    NULL AS app_channel,
+    metadata_app_version AS app_display_version,
+    NULL AS architecture,
+    NULL AS device_manufacturer,
+    device AS device_model,
+    NULL AS telemetry_sdk_build,
+    first_seen_date,
+    submission_date = first_seen_date AS is_new_profile,
+    NULL AS n_metrics_ping,
+    NULL AS days_sent_metrics_ping_bits,
+    NULL AS uri_count,
+    default_browser AS is_default_browser,
+    'Focus Android' AS normalized_app_name
```

Comment:
Focus Android doesn't have `clients_last_seen_joined` since it doesn't have `baseline_clients_last_seen`